libs/ledgerjs/packages/errors/src/index.ts

```diff
@@ -44,6 +44,7 @@ export const DeviceExtractOnboardingStateError = createCustomErrorClass(
 export const DeviceOnboardingStatePollingError = createCustomErrorClass(
   "DeviceOnboardingStatePollingError",
 );
+export const ExchangeTimeoutError = createCustomErrorClass("ExchangeTimeoutError");
```

Comment:
"Exchange" naming is a bit ambiguous (it could refer to the Exchange app for swapping crypto), could it be renamed to something more Transport related like MessageExchangeTimeoutError or something like that a bit more explicit?

Reply:
yeah you are right !